src/main/java/mate/academy/hibernate/relations/service/impl/MovieServiceImpl.java

```diff
@@ -1,16 +1,22 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.MovieDao;
+import mate.academy.hibernate.relations.dao.impl.MovieDaoImpl;
 import mate.academy.hibernate.relations.model.Movie;
 import mate.academy.hibernate.relations.service.MovieService;
+import mate.academy.hibernate.relations.util.HibernateUtil;
 
 public class MovieServiceImpl implements MovieService {
+    private MovieDao movieDao = new MovieDaoImpl(HibernateUtil.getSessionFactory());
```

Comment:
Direct instantiation of 'MovieDaoImpl' inside the service layer undermines the principle of Dependency Injection. Instead, pass 'MovieDao' as a constructor parameter to make testing easier and decouple the service from the specific implementation.